fastjet/internal/predicates/orientation.go

```diff
@@ -8,6 +8,8 @@ import (
 	"fmt"
 	"math/big"
 
+	"math"
```

Comment:
remove empty line 10. `"math"` is a stdlib package, it should be part of the stdlib import block.